### PR TITLE
[NIFI-3154] display ellipsis when text is sufficiently long in connec…

### PR DESCRIPTION
**Pull request by scottyaslan**
Thank you for submitting a contribution to Apache NiFi.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [ ] Does your PR title start with NIFI-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [ ] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed via mvn -Pcontrib-check clean install at the root nifi folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] If applicable, have you updated the LICENSE file, including the main LICENSE file under nifi-assembly?
- [ ] If applicable, have you updated the NOTICE file, including the main NOTICE file found under nifi-assembly?
- [ ] If adding new Properties, have you added .displayName in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

…tion details and connection configuration dialogs

### CODE REVIEW

**Comment by scottyaslan** [comment]
This PR addresses the long processor name/type, input/output port, and funnel values in the 'Connection Configuration' and 'Connection Details" dialogs. It adds ellipsis to the 'From <source>' and the 'To <destination>' text values for each type of <source> and <destination> types but leaves the 'Within Group' values as non-ellipsis since having really long names there does not make the dialog unreadable. @moranr Please take a look and let me know if you have any other suggestions. Thanks!

**Comment by moranr** [comment]
@scottyaslan this looks good. I put in a few long names in various cases and it is behaving as you describe.

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-connection-configuration.js [hunk 1]
```diff
@@ -189,7 +189,7 @@ nf.ConnectionConfiguration = (function () {
 
             // populate the port information
             $('#input-port-source').show();
-            $('#input-port-source-name').text(inputPortName);
+            $('#input-port-source-name').text(inputPortName).addClass('ellipsis').attr('title', inputPortName);
```
Comment (mcgilman) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-connection-configuration.js]: Can the ellipsis class for all fields changed in this file be applied directly to the element in the corresponding JSP?

**Comment by mcgilman** [comment]
Thanks @scottyaslan! This has been merged to master.

**Pull request closed without merging**